README.md

```diff
@@ -326,86 +326,9 @@ onLCP(sendToAnalytics);
 
 ### Send the results to Google Analytics
 
-Google Analytics does not support reporting metric distributions in any of its built-in reports; however, if you set a unique dimension value (in this case, the metric `id`, as shown in the examples below) on every metric instance that you send to Google Analytics, you can create a report yourself using the [Google Analytics Reporting API](https://developers.google.com/analytics/devguides/reporting) and any data visualization library you choose.
+Google Analytics does not support reporting metric distributions in any of its built-in reports; however, if you set a unique dimension value (in this case, the `metric_id`, as shown in the examples below) on every metric instance that you send to Google Analytics, you can create a report yourself using the [Google Analytics Reporting API](https://developers.google.com/analytics/devguides/reporting) and any data visualization library you choose.
```

Comment:
The Reporting API is for UA only. The Data API is the API for GA4, which is still in alpha/beta: https://developers.google.com/analytics/devguides/reporting/data/v1

Also it's  not clear to me whether "unique dimension value" is the right terminology given that this is going to be GA4 only; it might be "unique event parameter. However, I'm not 100% sure how you get those values via the API. It might still require creating a custom dimension in the UI, which is then defined as a custom event parameter value (again, not 100% sure).

Alternatively, we could just not recommend the API at all and stick to the recommendation of using BigQuery...

Reply:
OK let's just remove this section.